CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ Note that this project **does not** adhere to [Semantic Versioning](https://semv
 
 ### Added
 
+- We added a feature to convert keywords that resemble MSC codes to their descriptions. [#12944](https://github.com/JabRef/jabref/issues/12944)
+- We added a tooltip to keywords that resemble Math Subject Classification(MSC) codes. [#12944](https://github.com/JabRef/jabref/issues/12944)
```

Comment:
swapped the entries, as the abbreviation explanation should go first. Moreover, a space comes after the term and before the bracket.

```suggestion
- We added a tooltip to keywords that resemble Math Subject Classification (MSC) codes. [#12944](https://github.com/JabRef/jabref/issues/12944)
- We added a feature to convert keywords that resemble MSC codes to their descriptions. [#12944](https://github.com/JabRef/jabref/issues/12944)
```